bundles/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/internal/ChromecastCommander.java

```diff
@@ -135,10 +135,19 @@ private void handlePlayUri(Command command) {
     private void handleControl(final Command command) {
         try {
             if (command instanceof NextPreviousType) {
-                // I can't find a way to control next/previous from the API. The Google app doesn't seem to
-                // allow it either, so I suspect there isn't a way.
-                logger.info("{} command not yet implemented", command);
-                return;
+                // Next is implemented by seeking to the end of the current media
+                if (command == NextPreviousType.NEXT) {
```

Comment:
Maybe it has to be moved beliw, after checking if app is running?

Reply:
Good catch - moved.